apis/python/notebooks/tutorial_soma_append_mode.ipynb

```diff
@@ -673,6 +686,444 @@
         "    print()\n",
         "    print(X.used_shape())"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "## Ingesting multiple datasets to a SOMA Experiment"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "Finally, we'll demonstrate combining multiple AnnDatas into one new experiment.\n",
+        "\n",
+        "The flow is pretty similar to the above:\n",
+        "\n",
+        "1. One call to `register_anndatas` or `register_h5ads` (passing all input AnnDatas/h5ads)\n",
+        "2. One call to `from_anndata`/`from_h5ad` *for each input AnnData*\n",
+        "\n",
+        "Here's a helper function to simulate multiple lab runs. As above, where we used `pbmc3k` to simulate Monday and Tuesday data, here we use `pbmc3k` to simulate multiple AnnData objects."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 15,
+      "metadata": {
+        "tags": []
+      },
+      "outputs": [],
+      "source": [
+        "def make_ad(when, scale, obs_id_suffix):\n",
+        "    ad = ad1.copy()\n",
+        "    ad.obs.index = [e.replace(\"-1\", obs_id_suffix) for e in ad.obs.index]\n",
+        "    ad.obs[\"when\"] = [when] * len(ad.obs)\n",
+        "    ad.X *= scale\n",
+        "    return ad\n",
+        "\n",
+        "new_ads = [\n",
```

Comment:
Sorry, I missed this before & didn't see it until now :( -- same comment as before -- not using the "new" terminology for atlas mode since they may well be ingesting a bunc of old data -- old/new isn't the point here; many-to-one is.

Let's say `ads` or `adatas` or whatever. And below: `for ad in ads`.